bootstrap/src/sun/nio/ch/lincheck/Injections.java

```diff
@@ -632,4 +634,24 @@ public static int getNextEventId(String type) {
     public static void setLastMethodCallEventId() {
         getEventTracker().setLastMethodCallEventId();
     }
+    
+    
+    private static Constructor<MethodHandles.Lookup> lookUpPrivateConstructor = null;
+    
+    public static Class<?> getClassForNameOrNull(String name) {
+        try {
+            return Class.forName(name);
+        } catch (ClassNotFoundException e) {
+            return null;
+        }
+    } 
+    
+    public static MethodHandles.Lookup trustedLookup(Class<?> callerClass) throws Exception {
```

Comment:
Please add a documentation explaining what this method does, and why it is needed.